code/modules/surgery/emergency_cardioversion_recovery.dm

```diff
@@ -0,0 +1,119 @@
+/datum/surgery/cardioversion
+	name = "Emergency Cardioversion Induction"
+	steps = list(/datum/surgery_step/incise, /datum/surgery_step/retract_skin, /datum/surgery_step/saw, /datum/surgery_step/clamp_bleeders,
+				 /datum/surgery_step/incise_heart, /datum/surgery_step/ventricular_electrotherapy, /datum/surgery_step/close)
+	possible_locs = list(BODY_ZONE_CHEST)
+
+/datum/surgery_step/ventricular_electrotherapy
+	name = "ventricular electrotherapy"
+	implements = list(/obj/item/twohanded/shockpaddles = 90, /obj/item/defibrillator = 75, /obj/item/inducer = 55, /obj/item/stock_parts/cell = 25) //Just because the idea of a new player using the whole magine to defib is hillarious to me
+	time = 50
+	repeatable = TRUE //So you can retry
+
+/datum/surgery_step/incise_heart/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	display_results(user, target, "<span class='notice'>You begin apply an electrical charge directly to the heart and across the body...</span>",
+		"[user] begins to make an incision in [target]'s heart.",
+		"[user] begins to make an incision in [target]'s heart.")
+	target.notify_ghost_cloning("Your heart is undergoing Emergency Cardioversion Induction!")
+	playsound(src, 'sound/machines/defib_charge.ogg', 75, 0)
+
+/datum/surgery_step/incise_heart/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	playsound(src, 'sound/machines/defib_zap.ogg', 75, 1, -1)
+	playsound(src, "bodyfall", 50, 1)
+	if(H.stat == DEAD)
```

Comment:
Reading the lines below, I assume this surgery is not meant to succeed on the living.
```suggestion
	if(H.stat != DEAD)
```

Reply:
Oops